samples/core/xgboost_training_cm/xgboost_training_cm.py

```diff
@@ -22,7 +22,7 @@
 import subprocess
 
 diagnose_me_op = components.load_component_from_url(
-    'https://raw.githubusercontent.com/kubeflow/pipelines/df450617af6e385da8c436628afafb1c76ca6c79/components/diagnostics/diagnose_me/component.yaml')
+    'https://raw.githubusercontent.com/kubeflow/pipelines/9b0e9efb382c499cef2750d71cb7b97cbd65cd07/components/diagnostics/diagnose_me/component.yaml')
```

Comment:
Since another fix has been merged
```suggestion
    'https://raw.githubusercontent.com/kubeflow/pipelines/566dddfdfc0a6a725b6e50ea85e73d8d5578bbb9/components/diagnostics/diagnose_me/component.py')
```